nose_watcher/nose_watcher.py

```diff
@@ -42,6 +46,18 @@ def get_commandline_arguments(self, argv=None):
         args = [a for a in argv if a != '--with-%s' % PLUGIN_NAME]
         return args
 
+    def options(self, parser, env):
+        """ Configure with command line option '--filetype'. """
+        Plugin.options(self, parser, env)
+        parser.add_option('--filetype', action='append',
+                          help='Specify additional filetypes to monitor.')
+
+    def configure(self, options, conf):
+        """ Get filetype option to specify additional filetypes to watch. """
+        Plugin.configure(self, options, conf)
+        if options.filetype:
+            self.filetypes = tuple(list(self.python_files) + options.filetype)
```

Comment:
This looks particularly bad. If we changed `self.filetypes` to a list, then we could just do `self.filetypes += options.filetypes`